src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts

```diff
@@ -138,9 +129,28 @@ export class GoToStickyScrollLine extends EditorAction2 {
 	}
 
 	runEditorCommand(_accessor: ServicesAccessor, editor: ICodeEditor) {
-		const stickyScrollController = StickyScrollController.get(editor);
-		if (stickyScrollController) {
-			stickyScrollController.goToFocused();
-		}
+		StickyScrollController.get(editor)?.goToFocused();
+	}
+}
+
+export class SelectEditor extends EditorAction2 {
+
+	constructor() {
+		super({
+			id: 'editor.action.selectEditor',
```

Comment:
I'm not the best person to ask but not sure the ideal name for this, should a contrib be registering unprefixed `editor.action.*` commands?